src/probability/ident_distrib.lean

```diff
@@ -283,4 +290,58 @@ end
 
 end ident_distrib
 
+section uniform_integrable
+
+open topological_space
+
+variables {E : Type*} [measurable_space E] [normed_add_comm_group E] [borel_space E]
+  [second_countable_topology E] {μ : measure α} [is_finite_measure μ]
+
+/-- This lemma is superceded by `mem_ℒp.uniform_integrable_of_ident_distrib` which only require
+`ae_strongly_measurable`. -/
+lemma mem_ℒp.uniform_integrable_of_ident_distrib' {ι : Type*} {f : ι → α → E}
```

Comment:
```suggestion
lemma mem_ℒp.uniform_integrable_of_ident_distrib_aux {ι : Type*} {f : ι → α → E}
```
With a prime in the name, it is not clear if this is another useful of some lemma. With an `aux`, it is clear that one should not use it.